### ISSUE TITLE
Custom button click last step form validation

### ISSUE THREAD

**Issue by Kovaloff** [BinarCode/vue-form-wizard#262]
Hello! I'm trying to async validate last step tab by clicking on custom button.... 
My Button:
`<button v-on:click="validateThirdTab" type="button" class="btn btn-md btn-warning">Get instructions</button>`

And validation method:
`validateThirdTab: function () {
                return new Promise((resolve, reject) => {

                    axios.post('/box/user-box-primary-email', {primary_email: this.model.primary_email})
                    .then(function (response) {
                        this.showLastStepInstructions = true;
                        resolve(true);
                    })
                    .catch(function (error) {
                        if (error.response) {
                            // The request was made and the server responded with a status code
                            // that falls out of the range of 2xx
                            reject(error.response.data.errors);

                        } else if (error.request) {
                            // The request was made but no response was received
                            // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
                            // http.ClientRequest in node.js

                            reject(error.message);

                        } else {
                            // Something happened in setting up the request that triggered an Error
                            console.log('Error', error.message);
                        }

                        reject('Something went wrong!');
                    });
                })
            },`

I will be grateful for any clue....

**Comment by jonalxh** [comment]
@Kovaloff Have you solved it? It is not validating for me the last steps.

**Comment by Kovaloff** [comment]
@jonalxh yes, in my case i added `:before-change` to the tab i needed to validate

```html
  <tab-content :before-change="validateThirdTab" title="title....." icon="">
```
